libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -417,7 +425,37 @@ private void InitializeModel()
             byte[] snapShotByteArray = Encoding.UTF8.GetBytes(content);
```

Comment:
Just trying to learn about this piece of code: I assume this is executed only once on load, and not on a per-turn basis right?

Reply:
Also, lines 424 and 425 read the file twice and use twice as much memory as needed. Can both be replaced by 

```
byte[] snapshotBytes=  File.ReadAllBytes(fullSnapShotFile);
```

Avoiding the binary -> string -> binary extra copying and memory

Reply:
Agree, will use ReadAllBytes()

Reply:
And yes, the model file was loaded only once per load, not per turn.